back-end/routes/users.js

```diff
@@ -62,6 +62,21 @@ router.post('/login', [
   }
 });
 
+/* GET users recommendations */
+router.get('/:id/recommendations', validateSelfJWT, async function(req, res, next) {
+  // veryify permission and retrieve ID.
+  const userID = await getUserIdFromToken(req, res);
```

Comment:
We have the userId in the path param and the only way to get up to this point if through validateSelfJWT. I think it should be save to just use the path param directly?

Reply:
Yeah that's fair and makes sense. Will update

Reply:
Updated